pkg/api/routes.go

```diff
@@ -104,9 +112,22 @@ func (rh *RouteHandler) SetupRoutes() {
 			rh.ListRepositories).Methods("GET")
 		g.HandleFunc("/",
 			rh.CheckVersionSupport).Methods("GET")
+		g.HandleFunc("/debug/pprof/",
+			pprof.Index).Methods("GET")
+		g.HandleFunc("/debug/pprof/cmdline",
+			pprof.Cmdline).Methods("GET")
+		g.HandleFunc("/debug/pprof/profile",
+			pprof.Profile).Methods("GET")
+		g.HandleFunc("/debug/pprof/symbol/",
+			pprof.Symbol).Methods("GET")
+		g.HandleFunc("/debug/pprof/trace",
+			pprof.Trace).Methods("GET")
 	}
 	// swagger docs "/swagger/v2/index.html"
 	rh.c.Router.PathPrefix("/swagger/v2/").Methods("GET").Handler(httpSwagger.WrapHandler)
+	// Zot Search Extension Router
+	rh.c.Router.PathPrefix("/query").Methods("GET", "POST").Handler(rh.searchHandler())
+	rh.c.Router.PathPrefix("/debug/pprof/heap").Methods("GET").Handler(pprof.Handler("heap"))
```

Comment:
Remove this, or add a "profiling: bool" Extensions param.
And add a if ... that

Reply:
Yes, I added for analysing the memory leak. I will remove this endpoints.